benchmark/src/main/java/io/prometheus/benchmark/ExemplarsBenchmark.java

```diff
@@ -0,0 +1,80 @@
+package io.prometheus.benchmark;
```

Comment:
The readme should be updated with the new numbers.

Reply:
done

Reply:
This comment still stands.